raiden/tests/unit/storage/migrations/test_v19_to_v20.py

```diff
@@ -0,0 +1,127 @@
+import json
+from datetime import datetime
+from pathlib import Path
+from unittest.mock import Mock, patch
+
+from raiden.storage.migrations.v19_to_v20 import upgrade_v19_to_v20
+from raiden.storage.serialize import JSONSerializer
+from raiden.storage.sqlite import SerializedSQLiteStorage, SQLiteStorage
+from raiden.tests.utils.factories import make_32bytes, make_address
+from raiden.tests.utils.mocks import MockRaidenService
+from raiden.utils.serialization import serialize_bytes
+from raiden.utils.upgrades import UpgradeManager
+
+
+def setup_storage(db_path):
+    storage = SQLiteStorage(str(db_path))
+
+    state_changes_file = Path(__file__).parent / 'data/v19_statechanges.json'
+    state_changes_data = json.loads(state_changes_file.read_text())
+    for state_change_record in state_changes_data:
+        storage.write_state_change(
+            state_change=json.dumps(state_change_record[1]),
+            log_time=datetime.utcnow().isoformat(timespec='milliseconds'),
+        )
+
+    # Add the v18 events to the DB
+    events_file = Path(__file__).parent / 'data/v19_events.json'
+    events_data = json.loads(events_file.read_text())
+    event_tuples = []
+    for event in events_data:
+        state_change_identifier = event[1]
+        event_data = json.dumps(event[2])
+        log_time = datetime.utcnow().isoformat(timespec='milliseconds')
+        event_tuples.append((
+            None,
+            state_change_identifier,
+            log_time,
+            event_data,
+        ))
+    storage.write_events(event_tuples)
+
+    chain_state_data = Path(__file__).parent / 'data/v19_chainstate.json'
+    chain_state = chain_state_data.read_text()
+    cursor = storage.conn.cursor()
+    cursor.execute(
+        """
+        INSERT INTO state_snapshot(identifier, statechange_id, data)
+        VALUES(1, 1, ?)
+        """, (chain_state,),
+    )
+    storage.conn.commit()
+    return storage
+
+
+def test_upgrade_v19_to_v20(tmp_path):
+    old_db_filename = tmp_path / Path('v19_log.db')
+    with patch('raiden.utils.upgrades.older_db_file') as older_db_file:
```

Comment:
I don't really understand what you are testing here. Can you add a comment for clarification? v19 to v19?

Reply:
This is returning `old_db_filename` and storing v19 as the old version. Similar to the other migrations... not sure what requires additional clarifications